lib/fog/compute/kubevirt/models/template.rb

```diff
@@ -26,17 +28,20 @@ def clone(options = {})
         def self.parse(object)
           metadata = object[:metadata]
           annotations = metadata[:annotations]
-          {
-            namespace: metadata[:namespace],
-            name: metadata[:name],
-            description: annotations[:description],
-            tags: annotations[:tags],
-            labels: metadata[:labels],
-            resource_version: metadata[:resourceVersion],
-            uid: metadata[:uid],
-            objects: object[:objects],
-            parameters: object[:parameters],
+          object = {
```

Comment:
A matter of taste, but I would rather do without defining object:
{
..
..
:description => annotations && annotations[:description],
:tags => annotations && annotations[:tags]
}.compact

Reply:
Done